packages/renderer/src/lib/network/NetworkColumnDriver.svelte

```diff
@@ -0,0 +1,14 @@
+<script lang="ts">
+import type { NetworkInfoUI } from './NetworkInfoUI';
+
+interface Props {
+  object: NetworkInfoUI;
+}
+
+let { object }: Props = $props();
+</script>
+
+<button
+  class="hover:cursor-pointer flex text-[var(--pd-table-body-text-highlight)] max-w-full overflow-hidden text-ellipsis">
+  {object.driver}
+</button>
```

Comment:
question (blocking): why using a button here? Will we have a onclick with a specific action?